### ISSUE TITLE
Just what i have been looking for 

### ISSUE THREAD

**Issue by TheJammie** [Trsak/esx_inventoryhud#65]
I have spent about the last hour looking for this just to find this and could not be happier 

Just chucked it into the server everything is working apart from the shops for some reason? using esx_shops all items are in the database as it was working but just the default menu type shops before i pushed this update out. Any ideas what could be causing it? 

No error in the server console itself 

![Screenshot_241](https://user-images.githubusercontent.com/45109415/92420478-72521c80-f16b-11ea-8774-061e45e5f799.png)

**Comment by Trsak** [comment]
https://github.com/Trsak/esx_inventoryhud/wiki/esx_shops-integration

**Comment by TheJammie** [comment]
So attempted this twice. 

Every time its done for some reason it fully stops the shop script map blips disappear ect ect. Do i have to be running a certain version?

**Comment by Trsak** [comment]
> So attempted this twice.
> 
> Every time its done for some reason it fully stops the shop script map blips disappear ect ect. Do i have to be running a certain version?

Yes version linked in that wiki page. Any errors on console regarding to esx_shop or esx_inventoryhud?

**Comment by TheJammie** [comment]
Not a single error in the console or client side which is fully throwing me off [
![Screenshot_242](https://user-images.githubusercontent.com/45109415/92421264-a29bba00-f16f-11ea-9e26-85848f41b49d.png)
![Screenshot_243](https://user-images.githubusercontent.com/45109415/92421267-a3345080-f16f-11ea-828f-75d217698ee1.png)

](url)

**Comment by Trsak** [comment]
Try to use latest version and let me know.

**Comment by 0pax** [comment]
Hey, I don't have the same problem but I do have 2 problems with this, probably just a dumb mistake by me, but can you tell me which lines I need to change in esx_shops/client/main.lua? I've tried it several times with little to no success (the circles in the shops dissapear),  and also which lines should I change in esx_policejob if you could help me. I have the newest version of esx_shops also.
